src/chi/components/data-table/data-table.scss

```diff
@@ -32,6 +32,14 @@ $bg-url: $bg-url + ' fill="%230262b9" fill-rule="evenodd"/%3E%3C/svg%3E';
   overflow-y: initial;
   width: 100%;
 
+  .default-view {
```

Comment:
I think the idea was to add generic utility classes like
`-screen--only`
`-print--only`
@jllr @dani-cl-madrid Thoughts?